src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { get } from 'lodash';
+import { isEmpty } from 'lodash';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import crypto from 'crypto';
```

Comment:
Should it be included in dependency then?

Reply:
`crypto` is a built-in Node module, it should not be include as dependency

Reply:
Ok, but the code will be running in browser environment and not in node. Please check that the production build would contain the package.

Reply:
It seems like webpack handles it - https://webpack.js.org/configuration/node/. I have not tested though.